README.md

```diff
@@ -1,5 +1,9 @@
 # wtf-fix-paths
 
+Get started via 
+
+`usethis::use_course(https://github.com/rstats-wtf/wtf-explore-libraries/archive/master.zip)`
```

Comment:
The same comment I just made on a similar snippet elsewhere applied here re: formatting as a code chunk and using the much shorter `OWNER/REPO` spec.